meetings.md

```diff
@@ -0,0 +1,65 @@
+# Meetings
+
+The purpose of this resource is to share details about how and when we
+meet.
+
+**Contents**
+- [Types](#types)
+  - [All Hands](#all-hands)
+    - [Standup](#standup)
+    - [Full](#full)
+  - [Holds](#holds)
+- [Scheduling](#scheduling)
+- [Hosting](#hosting)
+- [Expectations](#expectations)
+
+## Types
+
+### All Hands
+
+All-hands meetings are our general sync-up. We all try our best to be
+there. We adjust regularity of meeting based on time of year, member
+workload (inside and outside co-op), and needs of organization-wide
+initiatives.
+
+:calendar: **Schedule:** We currently meet every 2 weeks, alternating between
+format of a short **standup** and a **full** meeting.
+
+#### Standup
+:clock10: **Duration:** 15 minutes
+
+This format involves...
+
+#### Full
+:clock10: **Duration:** 60 minutes
+
+This format involves...
+
+### Holds
+
+Holds are unclaimed spots in our shared schedule that we try to leave
+open. They could be used for:
+
+- Discussions requiring the whole organization
+- Last-minute or urgent meetings, for which a new scheduling is
+  difficult
+
+## Scheduling
+
+See the [_"Scheduling a meeting"_ protocol][scheduling-protocol].
+
+## Hosting
+
+See the [_"Hosting a meeting"_ protocol][hosting-protocol].
+
+   [scheduling-protocol]: /protocols.md#scheduling-a-meeting
+   [hosting-protocol]: /protocols.md#hosting-a-meeting
+
+## Expectations
```

Comment:
We could take this from the december retreat doc, but I think actually we don't have these yet. So maybe this is premature?

Reply:
Good point. Could we have a practice of leaving placeholders like this in skeleton of doc? I like to idea of leaving intended bits in the public face that we can be self-conscious about, to speak to aspirations even if not current state. Feels like it embodies "living document" ethos, and points to intentions in a way that casual readers can see (e.g., not navigating into our internals tracker for the issue mentioning it)

Reply:
Excessive placeholders doesn't feel living doc moreso "house still under construction (potentially abandoned)" to me, but I don't feel strongly about it in this file

Reply:
will remove :+1: